lisa/sut_orchestrator/azure/credential.py

```diff
@@ -114,6 +118,9 @@ def __init__(
             self._client_id = runbook.client_id
             self._log.debug(f"Use defined client id: {self._client_id}")
             os.environ["AZURE_CLIENT_ID"] = self._client_id
+        if runbook.allow_all_tenants:
```

Comment:
one line is enough: `self._allow_all_tenants = runbook.allow_all_tenants`. It doesn't like other values, which supports env.